src/WorksomeRectorConfig.php

```diff
@@ -50,6 +51,9 @@ public static function setup(RectorConfig $rectorConfig): void
         // PHP 8.0 set rules
         $rectorConfig->rule(ClassPropertyAssignToConstructorPromotionRector::class);
 
+        // PHP 8.1 set rules
```

Comment:
What's the benefit of this line and line 51?

Reply:
I guess nothing, I just copied the one above 🤷🏻 Lol

Reply:
Why have any? If we're that intent on grouping, we could split by declarative methods? Could be wrong here, I just think they'll probably get ignored by somebody at some point anyway

Reply:
Updated.